### PR TITLE
chore : added contribution and code of conduct guidelines

### PR DESCRIPTION
**Pull request by Ayush7614**
This PR Contains 
* Added Contribution Guidelines
* Added Code of Conduct
* We follow special code of conduct for our participants that is contributor covenant and it is available on [https://www.contributor-covenant.org/]

### CODE REVIEW

**Comment by Ayush7614** [comment]
cc @nehagup

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    * Follow all contribution guidelines
+    * Passes all checks and has no conflicts.
+    * Has a well-written title and message that briefly explains your proposed changes.
+
+## How to Report Bugs, Provide Feedback or Request Features?
+
+We welcome all kinds of bug reports, user feedback and feature requests!
+
+We've created some issue templates to assist you in this. Please use them to create [a new issue](https://help.github.com/en/github/managing-your-work-on-github/creating-an-issue) in the relevant project's repository.
```
Comment (nehagup) [inline CONTRIBUTING.md:36]: The URL should point to creating a new issue on this repository. Something like https://github.com/keploy/ui/issues/new/choose

**Review by nehagup — CHANGES_REQUESTED** [review]
Rest LGTM

File: CONTRIBUTING.md (hunk 1 above)
Comment (Ayush7614) [reply CONTRIBUTING.md]: Done @nehagup please review

**Comment by Ayush7614** [comment]
cc @nehagup

**Pull request merged**